packages/portfolio-contract/src/portfolio.flows.ts

```diff
@@ -466,6 +476,7 @@ const stepFlow = async (
             orch,
             'agoric',
             kit,
+            portfolioTrace,
```

Comment:
we can/should be more specific here, yes?

```suggestion
            indexTrace,
```